.github/release.yml

```diff
@@ -11,7 +11,10 @@ changelog:
         - refactoring
     - title: 📔 Documentation
       labels:
-        - docs
+        - documentation
     - title: 🔨 Dependency Upgrades
       labels:
         - dependencies
+    - title: 💥 Breaking Changes
+      labels:
+        - BREAKING
```

Comment:
Would be great if Renovate could mark major dependency updates automatically as breaking, right?

Reply:
Good idea alternative workflow could be that when a user reviews a major change he manually adds the breaking label. (That is the way i would do it as i especially look if there are breaking changes for the application in the release notes)
As for normal changes the author needs to think about if the change is breaking this might be ok too.